### ISSUE TITLE
[9.3.1] no carriage return in email notifications with rich text disabled

### ISSUE THREAD

**Issue by arengore** [glpi-project/glpi#4664]
* Steps to reproduce (which actions have you made) :

setup --> general --> assistance --> Use rich text for helpdesk --> No

* Expected result:

on description field, multilines tickets be correctly formatted also in email notifications

* Actual result :

tickets opened directly from helpdesk (and the same for ticket opened with email) are without carriage return when notified via email.

For example a description like this:

> This is a
> multiline description

when notified via email becomes:

> This is a multiline description

* Your GLPI setup (you can find it in Setup > General menu, System tab) :
```

Information about system installation and configuration
--
[code]   GLPI 9.3.1 ( => /var/www/site.com) Installation mode: TARBALL

Server
 
Operating system: Linux mail.site.com 4.9.0-8-amd64 #1 SMP Debian 4.9.110-3+deb9u4 (2018-08-21) x86_64
PHP 7.2.9-1+0~20180910100512.5+stretch~1.gbpdaac35 fpm-fcgi (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apcu,
	calendar, cgi-fcgi, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, igbinary, imap, json, ldap,
	libxml, mbstring, memcached, msgpack, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, sockets,
	sodium, standard, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zlib)
Setup: max_execution_time="600" memory_limit="256M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="30M" 
Software: Apache ()
	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36
Server Software: mariadb.org binary distribution
	Server Version: 10.3.9-MariaDB-1:10.3.9+maria~stretch-log
	Server SQL Mode: 
	Parameters: glpi@127.0.0.1/glpi
	Host info: 127.0.0.1 via TCP/IP
	
mysqli extension is installed
ctype extension is installed
fileinfo extension is installed
json extension is installed
mbstring extension is installed
zlib extension is installed
curl extension is installed
gd extension is installed
simplexml extension is installed
xml extension is installed
ldap extension is installed
imap extension is installed
Zend OPcache extension is installed
APCu extension is installed
xmlrpc extension is installed
CAS extension is installed
Database version seems correct (10.3.9) - Perfect!Database version seems correct (10.3.9) - Perfect!
OK/var/www/site.com/files/_log : OK
OK/var/www/site.com/config : OK
OK/var/www/site.com/files : OK
OK/var/www/site.com/files/_dumps : OK
OK/var/www/site.com/files/_sessions : OK
OK/var/www/site.com/files/_cron : OK
OK/var/www/site.com/files/_graphs : OK
OK/var/www/site.com/files/_lock : OK
OK/var/www/site.com/files/_plugins : OK
OK/var/www/site.com/files/_tmp : OK
OK/var/www/site.com/files/_cache : OK
OK/var/www/site.com/files/_rss : OK
OK/var/www/site.com/files/_uploads : OK
OK/var/www/site.com/files/_pictures : OK


Libraries
 
htmLawed version 1.2.4 in (/var/www/site.com/lib/htmlawed)
SimplePie version 1.5.2 in (/var/www/site.com/vendor/simplepie/simplepie/library)
TCPDF version 6.2.17 in (/var/www/site.com/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/site.com/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/site.com/vendor/true/punycode/src)
iacaml/autolink in (/var/www/site.com/vendor/iamcal/lib_autolink)
sabre/vobject in (/var/www/site.com/vendor/sabre/vobject/lib)
phpCas version 1.3.3 in (/usr/share/php)


Notifications
 
Way of sending emails: SMTP+TLS (ticket@127.0.0.1)


Plugins list
 
	gdrive               Name: GDrive                         Version: 1.1.0      State: Enabled

```

A sample html source for email with rich text disabled:

```
Content-Type: text/plain; charset=us-ascii

=-=-=-= To answer by email, write above this line =-=-=-=

Description: This is a
multiline description

-- 
Support Test
Automatically generated by GLPI

=_=_=_= To answer by email, write under this line =_=_=_=

--b1_k0WzyejPTu6QfmKIM42v2kX78qHAd3IaN2i1KCDCt7w
Content-Type: text/html; charset=utf-8
Content-Transfer-Encoding: 8bit

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
                        'http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd'><html>
                        <head>
                         <META http-equiv='Content-Type' content='text/html; charset=utf-8'>
                         <title>[TICKET #0000263] New ticket new ticket</title>
                         <style type='text/css'>
                           
                         </style>
                        </head>
                        <body>
=-=-=-= To answer by email, write above this line =-=-=-=
<br><br><p>Description: This is a
multiline description</p><p> </p><br><br>-- 
<br>Support Test<br>Automatically generated by GLPI<br><br>
=_=_=_= To answer by email, write under this line =_=_=_=
<br><br>
</body></html>
```

and the source with rich text enabled (correct visualization):

```
=-=-=-= To answer by email, write above this line =-=-=-=
<br><br><p>Description: <p>This is a</p><p>multiline description</p></p><p> </p><br><br>-- 
<br>Support Test<br>Automatically generated by GLPI<br><br>
=_=_=_= To answer by email, write under this line =_=_=_=
<br><br>
</body></html>
```

On 9.3.0 version this don't happen.

**Comment by trasher** [comment]
Duplicates https://github.com/glpi-project/glpi/issues/4623

**Comment by ppenin** [comment]
Is there any fix for this issue still remaining in 9.3.1 ?
